core/ui/KeyboardShortcuts/switcher.tid

```diff
@@ -2,4 +2,4 @@ title: $:/core/ui/KeyboardShortcuts/switcher
 tags: $:/tags/KeyboardShortcut
 key: ((layout-switcher))
 
-<$action-sendmessage $message="tm-show-switcher" switch="layout"/>
\ No newline at end of file
+<$action-sendmessage $message="tm-show-switcher" switch="layout" maskClosable=yes/>
```

Comment:
The convention is to have double quotes for strings in the TW core, to be consistent `maskClosable="yes"`. If we use it consistently, there is a higher chance that users also use them. It causes less issues.